lib/rspec/rails/view_path_builder.rb

```diff
@@ -0,0 +1,53 @@
+module RSpec
+  module Rails
+    # Builds paths for view specs using a particular route set.
+    class ViewPathBuilder
+      def initialize(route_set)
+        @routes = route_set.routes
+        self.class.send(:include, route_set.url_helpers)
```

Comment:
Does this always re-add the route helpers to the `ViewPathBuilder` class? I'm a little tired, I think I am missing something. If this uses `extend` on the object does it accomplish the same thing without affecting `ViewPathBuilder`?

```ruby
self.extend route_set.url_helpers
```